### DISCUSSION TITLE
harp.gl, react, and webpack

### DISCUSSION THREAD

**Discussion by frobones** [heremaps/harp.gl#2056] [Q&A] [answered]
Hi all,

I am by no means an expert when it comes to web development, when toying with react I've used `npx create-react-app my-app`. When toying with harp.gl, I've used `npm init @here/harp.gl-app`. However, now I need to use harp.gl inside react and messing with webpack to get the web workers to render the map tiles is proving real challenging for me.

So my question is, are there any "bootstrap" commands that can get harp.gl going in an react environment or any tutorials on how to get this going? Or am I going to have to buckle down and understand webpack to move forward?

**Comment by FraukeF** [comment, marked as answer]
Hi @frobones,
unfortunately we have no react+harp.gl bootstrap available at this point, but there is a bootstrap project done by another harp.gl user, maybe this helps: https://github.com/PanetoneCake/HarpGL-React.JS

**Reply by nzjony** [reply]
@frobones, I know you have seen [this](https://github.com/slaylines/harp.gl-react-example/issues/4#issuecomment-761300068), but I put two extra repos, incase anyone else is interested: check out: https://github.com/sainap/harpgl-samples/tree/main/3d-globe done by @sainap and https://github.com/slaylines/harp.gl-react-example

**Reply by frobones** [reply]
Thanks @FraukeF and @nzjony, these repos worked great for my needs.
